.github/labeler.yml

```diff
@@ -0,0 +1,2 @@
+automerge:
+- all: ['files/build/**/*']
```

Comment:
>files/build/**/* [](start = 9, length = 16)

Do you mean `files/build/versions/**` ?

Reply:
Yes, but not supported.
I want to add the label when only change in files/build/versions/**/* or files/build/versions/*, and not other change.
See https://github.com/actions/labeler.

all: match ALL globs against ALL changed paths

Reply:
Qi, it works as expected, fixed, thanks.